code/modules/projectiles/guns/mounted.dm

```diff
@@ -1,12 +1,11 @@
 /obj/item/weapon/gun/energy/gun/advtaser/mounted
 	name = "mounted taser"
 	desc = "An arm mounted dual-mode weapon that fires electrodes and disabler shots."
-	icon_state = "armcannon"
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "taser"
+	item_state = "armcannonstun4"
```

Comment:
![arm laser](https://cloud.githubusercontent.com/assets/8345184/13369244/61dc504c-dca7-11e5-9ff2-94b0475dd269.png)
![arm taser](https://cloud.githubusercontent.com/assets/8345184/13369245/61deb85a-dca7-11e5-83ae-847d813fad5b.png)
i have to admit these are pretty kickass
but does this mean the inhands are still the old blue ones? also you may want to test this, i remember trying to bypass the whole inhand charge meter thing and it was really buggy, and often the item would turn invisible entirely.